api/handler/v1beta1/appeal.go

```diff
@@ -195,6 +195,33 @@ func (s *GRPCServer) RevokeAppeal(ctx context.Context, req *guardianv1beta1.Revo
 	}, nil
 }
 
+func (s *GRPCServer) RevokeAppeals(ctx context.Context, req *guardianv1beta1.RevokeAppealsRequest) (*guardianv1beta1.RevokeAppealsResponse, error) {
+	filters := &domain.ListAppealsFilter{}
+	filters.AccountIDs = req.GetAccountIds()
+	filters.ProviderTypes = req.GetProviderTypes()
+	filters.ProviderURNs = req.GetProviderUrns()
+	filters.ResourceTypes = req.GetResourceTypes()
+	filters.ResourceURNs = req.GetResourceUrns()
```

Comment:
maybe like this more be straightforward
```suggestion
	filters := &domain.ListAppealsFilter{
	    AccountIDs : req.GetAccountIds()
	    ProviderTypes : req.GetProviderTypes()
	    ProviderURNs : req.GetProviderUrns()
	    ResourceTypes : req.GetResourceTypes()
	    ResourceURNs : req.GetResourceUrns()
	}
```